resources/Dockerfile

```diff
@@ -20,7 +20,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 # Install all CI dependencies
 RUN apt-get update \
         && apt-get -yq upgrade \
-        && apt-get install -yq \
+        && apt-get install --no-install-recommends -yq \
```

Comment:
This is probably what is causing your failure - you've got a package missing that the tests rely on.